vms/platformvm/config/execution_config.go

```diff
@@ -0,0 +1,61 @@
+// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package config
+
+import (
+	"encoding/json"
+
+	"github.com/ava-labs/avalanchego/utils/units"
+)
+
+const (
+	blockCacheSize               = 64 * units.MiB
+	txCacheSize                  = 128 * units.MiB
+	transformedSubnetTxCacheSize = 4 * units.MiB
+
+	validatorDiffsCacheSize = 2048
+	rewardUTXOsCacheSize    = 2048
+	chainCacheSize          = 2048
+	chainDBCacheSize        = 2048
+	checksumsEnabled        = false
+)
+
+// ExecutionConfig provides execution parameters of PlatformVM
+type ExecutionConfig struct {
+	BlockCacheSize               int  `json:"blockCacheSize"`
+	TxCacheSize                  int  `json:"txCacheSize"`
+	TransformedSubnetTxCacheSize int  `json:"transformedSubnetTxCacheSize"`
+	ValidatorDiffsCacheSize      int  `json:"validatorDiffsCacheSize"`
+	RewardUTXOsCacheSize         int  `json:"rewardUTXOsCacheSize"`
+	ChainCacheSize               int  `json:"chainCacheSize"`
+	ChainDBCacheSize             int  `json:"chainDBCacheSize"`
+	ChecksumsEnabled             bool `json:"checksums-enabled"`
```

Comment:
as noted by @najeal this is the only non-camel case tag. @StephenButtolph wonder if we could risk breaking backward compatibility to ensure uniformity.

Reply:
hmmm `checksums-enabled` hasn't been released yet - but I'm pretty sure we use dash separated names for all of our configs (and camelCase for all the APIs).

@najeal where do you see us using camelCase for consuming configs from the user?

Reply:
I have seen it for example in:
`network/config.go`
`node/config.go`
`utils/logging.go`
...

I think the only place I have seen `-` is for the VM config in `vms/avm/vm.go`. All other structs to unmarshal are camelCase excepted this one.

Reply:
I switched camelCase to dash in new commit